reference/5.1/Microsoft.PowerShell.Core/About/about_Preference_Variables.md

```diff
@@ -169,11 +169,10 @@ Cmdlets and functions that might pose a risk to the system have a **Confirm**
 parameter that you can use to request or suppress confirmation for a single
 command.
 
-Because most cmdlets and functions use the default risk value,
-**ConfirmImpact**, of **Medium**, and the default value of `$ConfirmPreference`
-is **High**, automatic confirmation rarely occurs. However, you can activate
-automatic confirmation by changing the value of `$ConfirmPreference` to
-**Medium** or **Low**.
+Most cmdlets and functions keep the default value of **Medium** for **ConfirmImpact**.
+`$ConfirmPreference` is set to **High** by default. Therefore, automatic
+confirmation rarely occurs. To extend automatic confirmation to more cmdlets and
```

Comment:
The phrase _automatic confirmation_ sounds misleading to me. It suggests to me `-Confirm:$false` instead of `-Confirm`. I think it'd be clearer something along the lines of _automatic confirmation prompt_ or _automatic interactive confirmation_. Just a remark, I won't change it in this PR unless requested.

Reply:
You're right, the current phrasing isn't clear enough. I think I would recommend something like:

```markdown
Most cmdlets and functions keep the default value of **Medium** for **ConfirmImpact**.
`$ConfirmPreference` is set to **High** by default. Therefore, it's rare that commands
automatically prompt for confirmation when users don't specify the **Confirm** parameter. To
extend automatic confirmation prompting to more cmdlets and functions, set the value of
`$ConfirmPreference` to **Medium** or **Low**.
```